integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/bool/BooleanSortAndRangePredicateIT.java

```diff
@@ -40,7 +40,7 @@
  * Tests sorting and ranging behaviour querying a boolean type field
  *
  * @see FieldSearchSortIT
- * @see RangeSearchPredicateIT
+ * @see RangeSearchPredicateLegacySyntaxIT
```

Comment:
Since this class is using the new API, maybe it should keep to reference `RangeSearchPredicateIT` and not the legacy one.